ui/components/app/flask/snap-install-warning/index.scss

```diff
@@ -5,6 +5,10 @@
     display: flex;
     align-items: flex-start;
     gap: 0 16px;
+
+    &:nth-child(n+3) {
```

Comment:
Can we use a `className`  here instead?  How sure are we that `n+3` is always going to be this element?  If we need to keep this, I'd love a comment as to what this element represents.